templates/manifest.json

```diff
@@ -1,106 +1,134 @@
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 256,
+                "timeoutSecs": 360
+            }
+        },
+        {
+            "name": "getting_started_typescript",
+            "label": "Start with TypeScript",
+            "category": "typescript",
+            "technologies": ["nodejs"],
+            "description": "Basic template for developing actors on the Apify platform. Using TypeScript and Node.js.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/getting_started_typescript.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 256,
+                "timeoutSecs": 360
+            }
+        },
+        {
+            "name": "getting_started_python",
+            "label": "Start with Python",
+            "category": "python",
+            "description": "Basic Python template for developing actors on the Apify platform.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/getting_started_python.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 256,
+                "timeoutSecs": 360
+            }
+        },
+        {
+            "name": "project_cheerio_crawler_js",
+            "label": "CheerioCrawler in JavaScript",
+            "category": "javascript",
+            "technologies": ["crawlee", "cheerio"],
+            "description": "Standard and up to date template for developing with Crawlee's CheerioCrawler.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_cheerio_crawler_js.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 512,
+                "timeoutSecs": 3600
+            },
+            "skipOptionalDeps": true
+        },
+        {
+            "name": "project_puppeteer_crawler_js",
+            "label": "PuppeteerCrawler in JavaScript",
+            "category": "javascript",
+            "technologies": ["crawlee", "puppeteer"],
+            "description": "Standard and up to date template for developing with Crawlee's PuppeteerCrawler.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_puppeteer_crawler_js.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 4096,
+                "timeoutSecs": 3600
+            }
+        },
+        {
+            "name": "project_playwright_crawler_js",
+            "label": "PlaywrightCrawler in JavaScript",
+            "category": "javascript",
+            "technologies": ["crawlee", "playwright"],
+            "description": "Standard and up to date template for developing with Crawlee's PlaywrightCrawler.",
```

Comment:
Standard and up-to-date template for developing with Crawlee's PlaywrightCrawler.